src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/DoNotUseCountWhenAnyCanBeUsedTests.Base.cs

```diff
@@ -65,5 +65,29 @@ protected async Task VerifyAsync(string methodName, string testSource, string fi
             }
         }
 
+        protected async Task VerifyAsync(string methodName, string testSource, string fixedSource, string extensionsSource, int line, int column)
+        {
+            try
+            {
+                await this.Verifier.VerifyAsync(
+                    methodName,
+                    new string[] { testSource, extensionsSource },
+                    new string[] { fixedSource, extensionsSource },
+                    line, column);
+            }
+            catch
+            {
+                this.Output.WriteLine($"{SourceProvider.CommentPrefix} Source code:{Environment.NewLine}{testSource}");
```

Comment:
Is it meant to be kept? This seems wrong to me.

Reply:
Yeah, good call, it was probably there for debugging purposes during development. I will remove it and all code related.

cc @paulomorgado

Reply:
@Jozkee,
If I was the one that did it, I don't recall it at the moment.